### DISCUSSION TITLE
Cyberduck Amazon S3 Listing Directory/ Failed + Questions

### DISCUSSION THREAD

**Discussion by JVanAlsenoy** [iterate-ch/cyberduck#15361] [Q&A]
Hello - I notice there is an open issue with title "Cyberduck Amazon S3 Listing Directory/ Failed" - I have the same issue.
I notice there was, in the previous issue, a request to activate the log and send the results but no further info is reported on the issue.  
(1) should I also do the same and send the log? 
(2) I am using the "DS3" solution offered by Cubbit.  I am very new to this and therefore am looking for 'direction'/help:
I did several tests of uploading, downloading etc - all went well until:
-- I started an upload for a large amount of data (folder structure - about 18GB worth of data) - I watched it for about 35mins; I then left my desk for some time. When I came back the computer (MacBook) was in 'sleep' mode (screen saver).  Could this be a cause of the problem? 
Is there other advice I need to be aware of?
Many thanks for your help -
Jacques Van Alsenoy

**Comment by tatze96** [comment]
I couldn't find the mentioned ticket. Can you tell me the ticket number so I can match them?

Regarding the listing issue:
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the listing issue, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15361 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

Regarding the sleep mode during the transfer process:
MacOS deactivates your network connection when the system enters sleep mode, which could interrupt the server connection and any active transfer processes within Cyberduck.

The screen saver usually only turns off the display to save energy but leaves any other hardware and software processes active, so it shouldn't interfere with Cyberduck.

**Reply by dkocher** [reply]
> MacOS deactivates your network connection when the system enters sleep mode, which could interrupt the server connection and any active transfer processes within Cyberduck.

When a transfer is in progress a sleep assertion is set which should assure macOS does not enter sleep mode.

**Comment by dkocher** [comment]
Relates to #15119.
